team/bundles/org.eclipse.compare/compare/org/eclipse/compare/contentmergeviewer/TextMergeViewer.java

```diff
@@ -1865,6 +1871,21 @@ protected ITokenComparator createTokenComparator(String line) {
 		return new TokenComparator(line);
 	}
 
+	/**
+	 * Creates an <code>IIgnoreWhitespaceContributor</code> which allows to hook
+	 * into the ignore whitespace logic in the compare viewer. Tool specific
+	 * implementations can overrule which whitespaces can be ignored and which not
+	 * (e.g. whitespaces in literals).
+	 *
+	 * @param document
+	 * @return a IIgnoreWhitespaceContributor which allows to overrule the platform
+	 *         based whitespace ignore logic in the compare view
+	 * @since 3.9
```

Comment:
Please add a sentence to the end of the return: `Default implementation doesn't supply a contributor`.

Reply:
done